Src/ILGPU/Runtime/Cuda/CudaNativeMethods.cs

```diff
@@ -248,6 +248,49 @@ enum Peer2PeerAttribute
         CU_DEVICE_P2P_ATTRIBUTE_NATIVE_ATOMIC_SUPPORTED = 3
     }
 
+    /// <summary>
+    /// Represents flags used to register host memory with the CUDA.
+    /// </summary>
+    [Flags]
+    public enum MemHostRegisterFlags
+    {
+        /// <summary>
+        /// If set, host memory is portable between CUDA contexts.
```

Comment:
I am not sure if we are allowed to include the driver API documentation here... I would feel better if we just briefly summarize in one line roughly what the content is.